plugins/lime-plugin-pirania/src/pages/renew.js

```diff
@@ -52,24 +60,26 @@ function Renew ({ goBack, loading, vouchers, getVoucherList }) {
 			.map(v => v.voucher);
 		setSelected(firstSelected);
 	}
+	console.log('renewed', renewed);
 	return (
 		<div>
-			<Box title={I18n.t('Renew')}>
+			<Box title={daysLeft + ' ' + I18n.t('days left')}>
```

Comment:
The daysLeft should be inside the translation, as sometimes you add it in front, and sometimes on the back of it.
Check "pluralization" on https://i18njs.com/
something like `I18n.t('%n days left')`